src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/ReportsService.kt

```diff
@@ -0,0 +1,88 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.service
+
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVPrinter
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.managerecallsapi.db.Recall
+import uk.gov.justice.digital.hmpps.managerecallsapi.db.RecallRepository
+import uk.gov.justice.digital.hmpps.managerecallsapi.domain.FileName
+import uk.gov.justice.digital.hmpps.managerecallsapi.service.ReportsService.Api.ReportCategory.WEEKLY_RECALLS_NEW
+import java.time.OffsetDateTime
+
+@Service
+class ReportsService(
+  @Autowired private val recallRepository: RecallRepository,
+) {
+
+  val csvMimeType = "text/csv"
+  private val headers = arrayOf(
+    "CUSTODY_TYPE_DESCRIPTION",
+    "CUSTODY_TYPE_AT_TIME_OF_RECALL_DESCRIPTION",
+    "FAMILY_NAME",
+    "FIRST_NAMES",
+    "NOMS_ID",
+    "PRISON_NUMBER",
+/*
+GENDER - n/a today; load in future?
+LICENCE_REVOKE_DATE - not stored on recall; save in future?
```

Comment:
I think we have this, its the date that the RecallNotification is sent I believe